docker.go

```diff
@@ -514,19 +516,71 @@ type DockerProvider struct {
 
 var _ ContainerProvider = (*DockerProvider)(nil)
 
+// or through Decode
+type TestContainersConfig struct {
+	Host      string `properties:"docker.host"`
+	TLSVerify int    `properties:"docker.tls.verify,default=0"`
+	CertPath  string `properties:"docker.cert.path,default="`
+}
+
 // NewDockerProvider creates a Docker provider with the EnvClient
 func NewDockerProvider() (*DockerProvider, error) {
-	client, err := client.NewEnvClient()
+	tcConfig := readTCPropsFile()
+	host := tcConfig.Host
```

Comment:
`TLSVerify` and `CertPath` seem to be ignored and not set on the `client`

Reply:
Yes, I'm not using them yet, although planning to do it in follow-up commits. Will mark this PR as draft meanwhile

Reply:
Used in https://github.com/testcontainers/testcontainers-go/pull/373/commits/af1986a417a441e72963e27b370c61d3053105f0